### ISSUE TITLE
no convergence

### ISSUE THREAD

**Issue by lzh990711** [hujie-frank/SENet#46]
Thank you for your share!
I use this network SE-BN-Inception.prototxt trained on ILSVRC2012 image data, but it can not converge. I implement this network on one GPU with memory 20G, so the batchsize is set to 128. The concrete parameter in the solver.prototxt are: base_lr(0.1),gamma(0.9),momentum(0.9),weight_decay(0.0005). I have also attempted to use serveral learning rate from 0.01 to 2, but all does not converge. The loss is 6.9 at the beginning, but after many iterations (e.g 100000,200000) the loss is still almost 6.9.
would you please give me some advices, thank you very much!!!
